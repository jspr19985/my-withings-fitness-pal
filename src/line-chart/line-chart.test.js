import LineChart from "./line-chart";
import { render } from '@testing-library/react';

describe("App", () => {
    test('renders Header', () => {
        const { container } = render(<LineChart />);
        expect(container).toMatchSnapshot();
    });
});
