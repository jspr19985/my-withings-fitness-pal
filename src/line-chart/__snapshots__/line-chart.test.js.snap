// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders LineChart 1`] = `
<div>
  <div
    class="VictoryContainer"
    style="height: 100%; width: 100%; user-select: none; pointer-events: none; position: relative;"
  >
    <svg
      height="300"
      role="img"
      style="pointer-events: all; width: 100%; height: 100%;"
      viewBox="0 0 450 300"
      width="450"
    >
      <g
        role="presentation"
      >
        <line
          role="presentation"
          shape-rendering="auto"
          style="stroke: white; fill: transparent; stroke-width: 1; stroke-linecap: round; stroke-linejoin: round;"
          vector-effect="non-scaling-stroke"
          x1="50"
          x2="400"
          y1="250"
          y2="250"
        />
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="50"
            y1="250"
            y2="251"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-0-tickLabels-0"
            x="50"
            y="269.55"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="middle"
              x="50"
            >
              01 AM
            </tspan>
          </text>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="400"
            x2="400"
            y1="250"
            y2="251"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-0-tickLabels-1"
            x="400"
            y="269.55"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="middle"
              x="400"
            >
              .001
            </tspan>
          </text>
        </g>
      </g>
      <g
        role="presentation"
      >
        <line
          role="presentation"
          shape-rendering="auto"
          style="stroke: white; fill: transparent; stroke-width: 1; stroke-linecap: round; stroke-linejoin: round;"
          vector-effect="non-scaling-stroke"
          x1="50"
          x2="50"
          y1="50"
          y2="250"
        />
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="49"
            y1="210"
            y2="210"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-1-tickLabels-0"
            x="39"
            y="213.55"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="end"
              x="39"
            >
              0.2
            </tspan>
          </text>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="49"
            y1="170"
            y2="170"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-1-tickLabels-1"
            x="39"
            y="173.55"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="end"
              x="39"
            >
              0.4
            </tspan>
          </text>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="49"
            y1="130"
            y2="130"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-1-tickLabels-2"
            x="39"
            y="133.55"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="end"
              x="39"
            >
              0.6
            </tspan>
          </text>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="49"
            y1="89.99999999999999"
            y2="89.99999999999999"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-1-tickLabels-3"
            x="39"
            y="93.54999999999998"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="end"
              x="39"
            >
              0.8
            </tspan>
          </text>
        </g>
        <g
          role="presentation"
        >
          <line
            role="presentation"
            shape-rendering="auto"
            style="stroke: white; fill: transparent; size: 1px;"
            vector-effect="non-scaling-stroke"
            x1="50"
            x2="49"
            y1="50"
            y2="50"
          />
          <text
            direction="inherit"
            dx="0"
            id="chart-axis-1-tickLabels-4"
            x="39"
            y="53.55"
          >
            <tspan
              dx="0"
              dy="0"
              style="stroke: white; font-size: 10px; font-family: Major Mono Display; letter-spacing: normal; padding: 10px; fill: #252525;"
              text-anchor="end"
              x="39"
            >
              1.0
            </tspan>
          </text>
        </g>
      </g>
      <g
        clip-path="url(#victory-clip-2)"
        style="border: 1px solid #ccc; width: 100%; height: 100%;"
      >
        <defs>
          <clippath
            id="victory-clip-2"
          >
            <rect
              height="200"
              vector-effect="non-scaling-stroke"
              width="350"
              x="50"
              y="50"
            />
          </clippath>
        </defs>
        <path
          d="M50,250L57,246L64,242L71,238L78,234L85,230L92,226L99,222L106,218L113,214.00000000000003L120,210L126.99999999999999,206L134,202L141,198L148,194L155,190L162,185.99999999999997L169,181.99999999999997L176.00000000000003,177.99999999999997L183.00000000000003,173.99999999999997L190.00000000000003,169.99999999999997L197.00000000000003,165.99999999999997L204.00000000000006,161.99999999999997L211.00000000000006,157.99999999999994L218.00000000000006,153.99999999999994L225.00000000000006,149.99999999999997L232.00000000000006,145.99999999999997L239.00000000000006,141.99999999999997L246.00000000000006,137.99999999999997L253.00000000000009,133.99999999999997L260.00000000000006,129.99999999999994L267.00000000000006,125.99999999999996L274.0000000000001,121.99999999999996L281.0000000000001,117.99999999999996L288.0000000000001,113.99999999999994L295.0000000000001,109.99999999999994L302.0000000000001,105.99999999999994L309.0000000000001,101.99999999999993L316.0000000000001,97.99999999999993L323.00000000000017,93.99999999999993L330.00000000000017,89.99999999999993L337.00000000000017,85.99999999999991L344.00000000000017,81.99999999999991L351.00000000000017,77.99999999999991L358.00000000000017,73.99999999999991L365.00000000000017,69.9999999999999L372.00000000000017,65.9999999999999L379.0000000000002,61.9999999999999L386.0000000000002,57.9999999999999L393.0000000000002,53.99999999999989L400,50"
          role="presentation"
          shape-rendering="auto"
          style="fill: transparent; stroke: #4D089A; stroke-width: 2;"
        />
      </g>
    </svg>
    <div
      style="z-index: 99; position: absolute; top: 0px; left: 0px; width: 100%; height: 100%;"
    >
      <svg
        height="300"
        style="overflow: visible; width: 100%; height: 100%;"
        viewBox="0 0 450 300"
        width="450"
      />
    </div>
  </div>
</div>
`;
